NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Not very familiar with colab, but I'm guessing <code>files</code> are fixtures that you are storing somewhere? Can you add some details on how the files are obtained for reproducibility?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>This will not be required going forward - I will code directly from my machine (should've probably started doing that way earlier!)</p>